### PR TITLE
Created datums can now take arguments for their new()

### PR DESCRIPTION
**Pull request by ShiftyRail**
[system]
pi% tested

I need this for reasons

### CODE REVIEW

File: code/modules/admin/verbs/debug.dm [hunk 1]
```diff
@@ -1244,7 +1244,18 @@ client/proc/check_convertables()
...
-	holder.marked_datum = new chosen
+	var/list/lst = list()
+	var/argnum = input("Number of arguments","Number:",0) as num|null
+	if(!argnum && (argnum!=0))
+		return
+
+	lst.len = argnum // Expand to right length
+
+	var/i
+	for(i = 1, i < argnum + 1, i++) // Lists indexed from 1 forwards in byond
```
Comment (Exxion) [inline code/modules/admin/verbs/debug.dm]: You can scrap the `lst.len = argnum` and `var/i` above and just do `for(var/i in 1 to argnum)` here and `lst += variable_set(src)` below

Comment (ShiftyRail) [reply code/modules/admin/verbs/debug.dm]: True, I copypasted this from elsewhere without looking at it in too much details.

**Comment by ShiftyRail** [comment]
@Exxion Done

**Comment by ShiftyRail** [comment]
This no longer works

**Pull request merged**